crates/app/Cargo.toml

```diff
@@ -5,6 +5,13 @@ authors = { workspace = true }
 edition = { workspace = true }
 
 [dependencies]
+anyhow = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 spin-locked-app = { path = "../locked-app" }
+
+[dev-dependencies]
+toml = { workspace = true }
+tempfile = { workspace = true }
```

Comment:
Is this used?

Reply:
It isn't now that i pulled in the factors tests. Removing!